internal/replay.go

```diff
@@ -528,16 +521,8 @@ func (r *ReplayCmd) replayUnavailabilityReasonExplanation(
 				" The calculated duration is: %s, calculated from: %s (time.Now) - %s (from)"+
 				" + %dm (start offset to ensure Replay covers the desired time window) %s."+
 				" Edit the Data Source and run Replay once again.",
-			replay.metricSource.Name,
-			replay.metricSource.Project,
-			expectedDuration.String(),
-			timeNow.Format(
-				flags.TimeLayout,
-			),
-			r.from.Format(flags.TimeLayout),
-			startOffsetMinutes,
-			offsetNotice,
-		)
+			replay.metricSource.Name, replay.metricSource.Project, expectedDuration.String(),
+			timeNow.Format(flags.TimeLayout), r.from.Format(flags.TimeLayout), startOffsetMinutes, offsetNotice)
```

Comment:
why did you change the formatting? please revert this

Reply:
No, I didn't :confused: